load-monitor/exporters/nginx_exporter

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+cat <<EOF >>/etc/nginx/sites-enabled/wazo
```

Comment:
Is it possible to move the content to another file.

`/etc/nginx-sites-enabled/wazo` is provided by the `wazo-nginx` package. If we modify it our modifications will be lost if/when `wazo-nginx` gets updated

Reply:
yes, makes sense.